src/components/SignUp/ActorInfo2.tsx

```diff
@@ -232,34 +232,14 @@ const ActorInfo2: React.FC<{
               </Form.Group>
             </Col>
           </Row>
-          {actorInfo2Gender !== '' && !actorInfo2Gender.includes('Cis') && (
-            <Row>
-              <Col lg="6">
-                <Form.Group>
-                  <CAGLabelSmaller>
-                    I would also be comfortable playing roles usually played by:{' '}
-                    <PrivateLabel />
-                  </CAGLabelSmaller>
-                  {genderRoles.map(g => (
-                    <Checkbox
-                      checked={isGenderRoleInGenderRoles(g)}
-                      fieldType="checkbox"
-                      key={`gender-chk-${g}`}
-                      label={g}
-                      name="actorInfo2GenderRoles"
-                      onChange={(e: any) =>
-                        genderRoleChange(e.currentTarget.checked, g)
-                      }
-                    />
-                  ))}
-                </Form.Group>
-              </Col>
-              <Col lg="6">
-                <Form.Group>
-                  <CAGLabelSmaller>
-                    I would be comfortable playing a character through all
-                    phases of their transition: <PrivateLabel />
-                  </CAGLabelSmaller>
+          <Row>
```

Comment:
You still have a ton of files in here undoing a lot of recent commits - you might want to go through each of these files and `git checkout master [filename]` just so we can make sure we're only keeping your new components and changes to the Profile page